src/api-reference/request/v4.get-started.markdown

```diff
@@ -15,131 +13,88 @@ Version 4.0 of Request API works only with the new [Authentication API](/api-ref
 
 * [Overview](#overview)
 * [Version](#version)
-* [Regional Availability](#regionalavailability)
-  * [With Curl](#curl)
+* [Process Flow](#process-flow)
+* [Products and Editions](#products-editions)
+* [Scope Usage](#scope-usage)
+* [Dependencies](#dependencies)
+* [Access Token Usage](#access-token-usage)
 * [Explore the API](#explore-api)
   * [Prerequisites](#prerequisites)
   * [Obtain an Access Token](#token)
-  * [Explore the API in JavaScript](#javascript)
-    * [Retrieve a User Access Token](#retrieve-token)
-    * [Post a request](#post-request)
 
 ## <a name="overview"></a>Overview
 
-The Request V4 API exposes four different resources:
+The Request v4 API exposes five different resources:
 
 Resource|Description
 ---|---
 Request|You can read, create, update or delete a Request and get the list of existing Requests.
 Workflow|You can perform action in the approval workflow of a Request (submit, approve, cancel...)
-Expense|You can read, create, update or delete an Expense, and get the list of expected Expenses for a specific Request.
+Expected Expense|You can read, create, update or delete an expected expense, and get the list of expected Expenses for a specific Request.
+Request Policy|You can get the list of existing Request policies.
 Travel Agency|You can get the description of a Travel Agency office.
 
 These resources are used to manage documents used for pre-spend authorizations within Concur Request.
 
 ## <a name="version"></a>Version
 
 * 4.0
-  * [Travel Request v4 - Endpoints](./v4.endpoints.html)
-  * [Travel Request v4 - Examples](./v4.examples.html)
-  * [Travel Request v4 - HTTP Status Codes](./v4.response-codes.html)
+  * Resources
+    * [Request](./v4.endpoints.request-resources.html)
+    * [Workflow](./v4.endpoints.workflow-resources.html)
+    * [Expected Expense](./v4.endpoints.expected-expense-resources.html)
+    * [Request Policy](./v4.endpoints.request-policy-resources.html)
+    * [Travel Agency](./v4.endpoints.travel-agency-resources.html)
+  * [Schema](./v4.endpoints.schemas.html)
+  * [HTTP Status Codes](./v4.response-codes.html)
 * [3.0](./v3.request.html)
 * [1.0](./v1.request.html)
 
-## <a name="regionalavailability"></a>Regional Availability
+## <a name="process-flow"></a>Process Flow
+![Process Flow for Request V4](./v4.request-process-flow.png)
```

Comment:
This image is not part of this PR, can you please add it?

Reply:
Done